charts/nd-common/templates/_datadog.tpl

```diff
@@ -28,6 +28,24 @@ ad.datadoghq.com/{{ .Chart.Name }}.instances: |-
   ]
 {{- end }}
 
+{{/*
+This is datalog logging configuration. We take the .Values.scrapeLogs and
+.Values.scrapeLogsProcessingRules map and convert into list of objects converted
+into json supported by datadog config. If source and service tag values not provided
+we add default values to it.
+*/}}
+
+{{- if and .Values.datadog.enabled .Values.datadog.scrapeLogs.enabled }}
```

Comment:
To test this out for basic function (not datadog function, but to make sure that the text is at least rendered), let's add some fake datadog settings into https://github.com/Nextdoor/k8s-charts/blob/main/charts/simple-app/ci/datadog-with-env-values.yaml and https://github.com/Nextdoor/k8s-charts/blob/main/charts/simple-app/ci/datadog-without-env-values.yaml that would cause this to get rendered... (oh and https://github.com/Nextdoor/k8s-charts/blob/main/charts/simple-app/values.local.yaml).. 

just makes it easier to catch errors if we are adding these annotations to our tests..

Reply:
fixed it